st2common/st2common/util/action_db.py

```diff
@@ -283,9 +283,9 @@ def serialize_positional_argument(argument_type, argument_value):
             argument_value = '1' if bool(argument_value) else '0'
         else:
             argument_value = ''
-    elif argument_type == 'list':
+    elif argument_type in [ 'array', 'list' ]:
```

Comment:
```suggestion
    elif argument_type in ['array', 'list']:
```

This should fix the Travis linting build.

Reply:
Done. Unfortunately the build still fails but this looks like it ran into some timeout. Let me know if anything else is needed.